src/components/TermsOfUsePage.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+<div class="grid-container body-search-content">
+  <h2>Terms of Use</h2>
+  <p>Do you accept?</p>
+  <form @submit.prevent>
+    <input type="button" value="No" class="button rounded alert" @click="rejectTermsOfUse">
+    <input type="button" value="Yes" class="button rounded" @click="acceptTermsOfUse">
```

Comment:
This would probably be more natural as a checkbox control, but we can deal with that once we get some feedback. At least the machinery is all in place now.